contrib/cirrus/runner.sh

```diff
@@ -199,8 +199,16 @@ function _run_altbuild() {
 }
 
 function _run_release() {
-    if bin/podman info |& grep -Eq -- '-dev'; then
-        die "Releases must never contain '-dev' in output of 'podman info'"
+    # TODO: These tests should come from code external to the podman repo.
+    # to allow test-changes (and re-runs) in the case of a correctible test
+    # flaw or flake at release tag-push time.  For now, the test is here
+    # given it's simlicity.
```

Comment:
Typo, simplicity. And I really don't understand this comment anyway, but trust that you do and it may help you in the future.

Reply:
got it.